modules/dnn/misc/onnx/opencv-onnx.pb.h

```diff
@@ -0,0 +1,5849 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: onnx.proto
+
+#ifndef PROTOBUF_onnx_2eproto__INCLUDED
+#define PROTOBUF_onnx_2eproto__INCLUDED
+
+#include <string>
+
+#include <google/protobuf/stubs/common.h>
+
+#if GOOGLE_PROTOBUF_VERSION < 3005000
+#error This file was generated by a newer version of protoc which is
+#error incompatible with your Protocol Buffer headers.  Please update
+#error your headers.
+#endif
+#if 3005000 < GOOGLE_PROTOBUF_MIN_PROTOC_VERSION
```

Comment:
opencv-caffe.proto and TensorFlow's `.proto` files were compiled using protobuf compiler 3.5.1. Could you please regenerate it for ONNX with the same version?


https://github.com/opencv/opencv/blob/2f0fc920dd51a3627779dcc7458abdbb09c933e6/modules/dnn/misc/tensorflow/graph.pb.h#L16